tests/integration/test_github.py

```diff
@@ -80,6 +80,12 @@ def test_pr_comments(self):
         assert pr_comments[0].comment.endswith("fixed")
         assert pr_comments[1].comment.startswith("LGTM")
 
+        comment_str = str(pr_comments[0])
+        assert comment_str.startswith(
+            "GithubPRComment(comment='@TomasTome...', author='lachmanfrantisek', created='"
+        ) and comment_str.endswith(")")
+        assert "edited" in comment_str
```

Comment:
I am not sure if these tests are useful, but why not. (I understand the coverage effect, but practically it doesn't gives us anything practical.)

Reply:
Up to you, it's just regarding the coverage, I can drop it.